go/vt/tabletserver/tabletenv/tabletenv.go

```diff
@@ -29,10 +29,27 @@ var (
 	WaitStats = stats.NewTimings("Waits")
 	// KillStats shows number of connections being killed.
 	KillStats = stats.NewCounters("Kills", "Transactions", "Queries")
-	// InfoErrors shows number of various non critical errors happened.
-	InfoErrors = stats.NewCounters("InfoErrors", "Retry", "DupKey")
 	// ErrorStats shows number of critial erros happened.
-	ErrorStats = stats.NewCounters("Errors", "Fail", "TxPoolFull", "NotInTx", "Deadlock", "Fatal")
+	ErrorStats = stats.NewCounters(
+		"Errors",
+		"OK",
```

Comment:
Can you please use the enums here instead?

(I understand that we're probably not going to change the names. But still, this way the connection is more clear where this list comes from :)